CLI/search/__main__.py

```diff
@@ -2,40 +2,78 @@
 import os
 import sys
 
-def main():
+def search_extension(path, ext):
+    """Searches for extensions in the path"""
+    output = []
+    for root, dirs, files in os.walk(path, topdown=True):
+        for file in files:
+            if file.endswith(ext):
+                path = os.path.join(root, file)
+                output.append(path)
+
+    return output
+
+
+def search_file(path, f):
+    """Searches for files in the path"""
     output = []
+    for root, dirs, files in os.walk(path, topdown=True):
+        for file in files:
+            if file == f:
+                path = os.path.join(root, file)
+                output.append(path)
+
+    return output
```

Comment:
I've added functions to search for a file/extension, as I used them a few time throughout the code, and to clean it up a bit